pulpcore/pulpcore/app/settings.py

```diff
@@ -206,6 +206,7 @@
     },
     'content': {
         'web_server': 'django',
+        'hostname': 'localhost:8000',
```

Comment:
I think you can access the original request in the serializer via `context`

Reply:
If the content app and the rest api are being served on different hosts, the context will contain the wrong hostname and port.

Reply:
Perhaps this settings should default to `None` and have viewset code inject the _host_ from the request when not specified here.  Most users will run the API and content app on the same box and would only need to set this when the content app is running on another box.  Less to keep up with.

Also, based on URL specifications, the setting should be named _host_ instead of _hostname_.  It can be either an IP or hostname and can optionally include the port.

Reply:
+1 @jortel - i'll rename it and change the default